webroot/js/debug_kit.js

```diff
@@ -1,25 +1,29 @@
-var baseUrl, toolbar;
+import './jquery.js';
+import {Toolbar} from './toolbar-app.js';
 
-var elem = document.getElementById('__debug_kit_app');
-if (elem) {
-  window.__debugKitId = elem.getAttribute('data-id');
-  window.__debugKitBaseUrl = elem.getAttribute('data-url');
-  window.__debugKitWebroot = elem.getAttribute('data-webroot');
-  elem = null;
-}
+( function( $ ) {
+    'use strict';
 
-$(document).ready(function() {
-  toolbar = new Toolbar({
-    toolbar: $('#toolbar'),
-    content: $('#panel-content-container'),
-    panelButtons: $('.panel'),
-    panelClose: $('#panel-close'),
-    keyboardScope : $(document),
-    currentRequest: __debugKitId,
-    originalRequest: __debugKitId,
-    baseUrl: __debugKitBaseUrl,
-    webroot: __debugKitWebroot,
-  });
+    let elem = document.getElementById( '__debug_kit_app' );
+    if( elem ) {
```

Comment:
Is this formatting the preferred standard for js now? `if() {` ? usually it's `if () {`

Reply:
We should stuck to as close as possible to what we use for PHP, so `if (..) {`.

Reply:
Indeed using basic eslint was also my plan.
But first I have to do the base work of actually refactoring the current jQuery mess into ES6 modules.
After the functional part is over we can then argue about what JS code styling we use...